ArcGISRuntimeSDKQt_CppSamples/Routing/FindClosestFacilityToMultipleIncidentsService/FindClosestFacilityToMultipleIncidentsService.cpp

```diff
@@ -0,0 +1,241 @@
+{
+  return m_mapView;
+}
+
+// Set the view (created in QML)
+void FindClosestFacilityToMultipleIncidentsService::setMapView(MapQuickView* mapView)
+{
+  if (!mapView || mapView == m_mapView)
+    return;
+
+  m_mapView = mapView;
+  m_mapView->setMap(m_map);
+
+  emit mapViewChanged();
+}
+
+void FindClosestFacilityToMultipleIncidentsService::createSymbols()
+{
+  // create picture marker symbol for facilities
+  m_facilitySymbol = new PictureMarkerSymbol(QUrl("https://static.arcgis.com/images/Symbols/SafetyHealth/FireStation.png"), this);
+  m_facilitySymbol->setWidth(30.0f);
+  m_facilitySymbol->setHeight(30.0f);
+
+  // create picture marker symbol for incidents
+  m_incidentSymbol = new PictureMarkerSymbol(QUrl("https://static.arcgis.com/images/Symbols/SafetyHealth/esriCrimeMarker_56_Gradient.png"), this);
+  m_incidentSymbol->setWidth(30.0f);
+  m_incidentSymbol->setHeight(30.0f);
+
+  // create simple line symbol for the route
+  m_routeSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle::Solid, Qt::blue, 2.0f, this);
+}
+
+void FindClosestFacilityToMultipleIncidentsService::createFeatureLayers()
+{
+  // create service feature table from feature server
+  m_facilitiesFeatureTable = new ServiceFeatureTable(QUrl("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0"), this);
+  // create feature layer from feature service table
+  m_facilitiesFeatureLayer = new FeatureLayer(m_facilitiesFeatureTable, this);
+  // apply the facilities symbol as the renderer for this layer
+  m_facilitiesFeatureLayer->setRenderer(new SimpleRenderer(m_facilitySymbol, this));
+
+  // create service feature table from feature server
+  m_incidentsFeatureTable = new ServiceFeatureTable(QUrl("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Incidents/FeatureServer/0"), this);
+  // create feature layer from feature service table
+  m_incidentsFeatureLayer = new FeatureLayer(m_incidentsFeatureTable, this);
+  // apply the incidents symbol as the renderer for this layer
+  m_incidentsFeatureLayer->setRenderer(new SimpleRenderer(m_incidentSymbol, this));
+
+  // connect to the doneLoading signal which calls the slot to set the view point geometry
+  connect(m_facilitiesFeatureTable, &ServiceFeatureTable::doneLoading, this, &FindClosestFacilityToMultipleIncidentsService::setViewpointGeometry);
+  connect(m_incidentsFeatureTable, &ServiceFeatureTable::doneLoading, this, &FindClosestFacilityToMultipleIncidentsService::setViewpointGeometry);
+
+  // load both service feature tables
+  m_facilitiesFeatureTable->load();
+  m_incidentsFeatureTable->load();
+
+}
+
+void FindClosestFacilityToMultipleIncidentsService::setupRouting()
+{
+  // connect to createDefaultParametersCompleted signal of the closest facility task
+  connect(m_task, &ClosestFacilityTask::createDefaultParametersCompleted, this, [this]
+          (QUuid, Esri::ArcGISRuntime::ClosestFacilityParameters defaultParameters)
```

Comment:
This argument is passed by const-reference in the API. this signature should match.

Also, the full namespace is not needed here.